app/developers/routes.py

```diff
@@ -654,6 +656,64 @@ def add_question_condition(grant_id: UUID, question_id: UUID, depends_on_questio
     )
 
 
+@developers_blueprint.route(
+    "/grants/<uuid:grant_id>/collections/questions/<uuid:question_id>/condition/<uuid:expression_id>",
+    methods=["GET", "POST"],
+)
+@is_platform_admin
+@auto_commit_after_request
+def edit_question_condition(grant_id: UUID, question_id: UUID, expression_id: UUID) -> ResponseReturnValue:
+    question = get_question_by_id(question_id)
+    expression = next(expression for expression in question.conditions if expression.id == expression_id)
```

Comment:
Feels like this might be nice in a small 🤏  - either an interface for get_expression or move this filtering to the Question model and have Question.get_expression()? :think]

Reply:
Yeah agree @samuelhwilliams - I'll do whichever we feel is best as a follow up

One thing I'd be keen on getting feeling sensible here is making sure that the expression id thats been passed in reasonably belongs to the question (which is why I didn't get it directly from the DB) - we could always just make most accessor methods have to provide the "context" you're getting them for (i.e the grant, or question in this case) but make it a get_expression method on the question model feels quite nice for now